config/packages/dev/storage.yaml

```diff
@@ -0,0 +1,10 @@
+services:
+    Aws\S3\S3Client:
+        arguments:
+            -   version: latest
+                region: '%env(S3_REGION)%'
+                endpoint: 'http://storage:9000'
```

Comment:
The endpoint should be dynamic, based on the environment, so an environment variable might be better than a dev specific file. Not sure, up to you

Reply:
I agree an env var might be a good solution but in production with a real S3 storage (not Minio server) we don't have to set a value to the `endpoint` parameter (Same with `use_path_style_endpoint`)

In my opinion it's better to let the parameter empty in other environment than dev 
And use env file to specify it on dev mode

Reply:
I agree an env var might be a good solution but in production with a real S3 storage (not Minio server) we don't have to set a value to the `endpoint` parameter (Same with `use_path_style_endpoint`)

In my opinion it's better to let the parameter empty in other environment than dev 
And use env file to specify it on dev mode